### PR TITLE
Pull request to setup workflow to deploy strapi to test/prod

### PR DESCRIPTION
**Pull request by parulmishra**
Pull request to setup workflow to deploy strapi to test/prod

### CODE REVIEW

File: .github/workflows/deploy-react-to-test-prod.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           core.exportVariable('ENV', env);
     - name: install oc
       run: |
-        curl -O https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.1/openshift-client-linux.tar.gz > oc.tar.gz
+        curl -O https://mirror.openshift.com/pub/openshift-v3/clients/3.11.232/linux/oc.tar.gz > oc.tar.gz
```
Comment (thegentlemanphysicist) [inline .github/workflows/deploy-react-to-test-prod.yaml:28]: good catch on this one

File: ansible/tasks/deploy_strapi_test_prod.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---  
+  - name: deploy {{ react_infra_name }} of PR {{ PR }} in {{ namespace }} namespace
```
Comment (thegentlemanphysicist) [inline ansible/tasks/deploy_strapi_test_prod.yaml:2]: should this be 'react_infra_name' or is there a 'strapi_infra_name'

File: openshift/templates/strapi/dc.yaml [hunk 3]
```diff
@@ -104,6 +104,15 @@ objects:
       test: false
       triggers:
         - type: ConfigChange
+        - imageChangeParams:
```
Comment (thegentlemanphysicist) [inline openshift/templates/strapi/dc.yaml:107]: Does this mean a re-deployment will be automatically triggered each time openshift detects a change in the relevant image?

**Review by thegentlemanphysicist — COMMENTED** [review]
Juat left a couple of questions

File: ansible/tasks/deploy_strapi_test_prod.yaml (hunk 2 above)
Comment (parulmishra) [reply ansible/tasks/deploy_strapi_test_prod.yaml:2]: Good catch.

File: openshift/templates/strapi/dc.yaml (hunk 3 above)
Comment (parulmishra) [reply openshift/templates/strapi/dc.yaml:107]: Yes, We should be following similar standards/triggers so I made it similar to react dc.yaml

Comment (thegentlemanphysicist) [reply openshift/templates/strapi/dc.yaml:107]: sounds good, just wanted to make sure I was reading it right

**Review by thegentlemanphysicist — DISMISSED** [review]

**Review by thegentlemanphysicist — APPROVED** [review]
Changes look good, though prod has not been successfully deployed yet.  I have added the jwt token secret, but I'm gettig the error: `cannot trigger a deployment for "strapi-prod" because it contains unresolved images` when I attempt to redeploy prod manually.

**Review by thegentlemanphysicist — APPROVED** [review]
Glad that this bug is fixed :)

**Pull request merged**